src/code/mapDrawing.js

```diff
@@ -93,15 +93,28 @@ export const getPopupBodyWithType = (portal, target) => {
 
 //** This function returns the appropriate image for a marker type */
 const getImageFromMarkerType = (type) => {
+    /* SCB: why not just cram these in a map type */
```

Comment:
We could do that with the implementation of the function, but I still like it as a function. You also still have to handle the `default:` case.

Reply:
if you like it as a function, leave it be.